src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private final double radius;
+    private final String color;
+    private final double areaCircle;
```

Comment:
color should be in Figure class. areaCircle is useless
```suggestion
    private double radius;
```